akka-cluster/src/main/resources/reference.conf

```diff
@@ -153,6 +153,11 @@ akka {
     # longer than this it could lead to removed nodes being re-added to the cluster
     prune-gossip-tombstones-after = 24h
 
+    # Try to limit the number of connections between datacenters. Used for gossip and heartbeating.
+    # This will not limit connections created for the messaging of the application.
+    # If the cluster does not span multiple data centers, this value has no effect.
+    cross-data-center-connections = 5
```

Comment:
I think we'll move that into a section like `multi-data-center` in my PR i have that section but let's fix that later btw